src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,43 @@
 
 class Animal {
   // write your code here
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
```

Comment:
use static for it

Reply:
We can't use static in this task without manipulation with lint, I don't know how do this. In general linter swears if I write static alive = [].

Reply:
Need add parser in .eslintrc.js
 "parser": "@babel/eslint-parser",
 
 Update - workflows/test.yml -  node-version to 14 
 add to package.json - devDependencies
 "@babel/eslint-parser": "^7.22.15",
  "@mate-academy/scripts": "^1.2.12", ( update version)